### ISSUE TITLE
Could you provide a  direct link of the dataset used in this paper? thanks~

### ISSUE THREAD

**Issue by fengxin619** [nachiket92/conv-social-pooling#6]

**Comment by rohanchandra30** [comment]
Yes please. Can you host the 6 datasets in your /raw folder on google drive or dropbox? Thanks

**Comment by nextfeixiag** [comment]
google driver is ok~ On 10/19/2018 08:08, Rohan Chandra wrote: Yes please. Can you host the 6 datasets in your /raw folder on google drive or dropbox? Thanks — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by nextfeixiag** [comment]
Or， could you privide the trainset.mat valset.mat testset.mat ? On 10/19/2018 08:19, LiuChuang wrote: google driver is ok~ On 10/19/2018 08:08, Rohan Chandra wrote: Yes please. Can you host the 6 datasets in your /raw folder on google drive or dropbox? Thanks — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by rohanchandra30** [comment]
How do you extract the latitudinal and longitudinal maneuvers from the data files?

**Comment by nachiket92** [comment]
Sorry for the late response. I'm not sure if I'm allowed to host the data. I'll try to include a line to download it in the pre-processing script.

**Comment by guoyage** [comment]
@nachiket92 First of all, congratulations for this fantastic work!  
I have downloaded all the zip files as well as the large csv file 
"Next_Generation_Simulation__NGSIM__Vehicle_Trajectories_and_Supporting_Data.csv" 
But the data needs to be processed to get the corresponding six files.
So I hope that you can provide download links for six files.
thank you for you help

**Comment by varunjammula** [comment]
https://data.transportation.gov/Automobiles/Next-Generation-Simulation-NGSIM-Vehicle-Trajector/8ect-6jqj
